<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="0" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2020-01-16T12:22:57Z" name="Suite" finished-at="2020-01-16T12:22:58Z" duration-ms="801">
    <groups>
    </groups>
    <test started-at="2020-01-16T12:22:57Z" name="loginTest" finished-at="2020-01-16T12:22:58Z" duration-ms="801">
      <class name="com.paralleltest.SignInPageTest">
        <test-method is-config="true" signature="setUpSuite()[pri:0, instance:com.paralleltest.SignInPageTest@17a7f733]" started-at="2020-01-16T12:22:56Z" name="setUpSuite" finished-at="2020-01-16T12:22:57Z" duration-ms="623" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpSuite -->
        <test-method signature="loginTestDBData(java.lang.String, java.lang.String)[pri:0, instance:com.paralleltest.SignInPageTest@17a7f733]" started-at="2020-01-16T12:22:58Z" name="loginTestDBData" data-provider="databaseTestData" finished-at="2020-01-16T12:22:58Z" duration-ms="0" status="SKIP">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.lang.NullPointerException]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.lang.NullPointerException
at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:49)
at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:131)
at org.testng.internal.Parameters.handleParameters(Parameters.java:706)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:49)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:37)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:924)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.NullPointerException
at com.paralleltest.utils.DBDataProvider.getDatabaseData(DBDataProvider.java:38)
at com.paralleltest.utils.DBDataProvider.databaseTestData(DBDataProvider.java:82)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:45)
... 10 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTestDBData -->
      </class> <!-- com.paralleltest.SignInPageTest -->
    </test> <!-- loginTest -->
  </suite> <!-- Suite -->
</testng-results>
